src/cargo/core/shell.rs

```diff
@@ -457,7 +457,7 @@ mod imp {
             let mut winsize: libc::winsize = mem::zeroed();
             // The .into() here is needed for FreeBSD which defines TIOCGWINSZ
             // as c_uint but ioctl wants c_ulong.
-            if libc::ioctl(libc::STDERR_FILENO, libc::TIOCGWINSZ.into(), &mut winsize) < 0 {
+            if libc::ioctl(libc::STDERR_FILENO, libc::TIOCGWINSZ, &mut winsize) < 0 {
```

Comment:
Note the comment right above this, which explains the `.into()`: it's platform-specific.

Reply:
Aha! I see